apps/web/src/app/api/checker/regions/_checker.ts

```diff
@@ -67,11 +55,18 @@ export const checker = async (request: Request, region: string) => {
     throw new Error("Could not parse request");
   }
 
-  const data = monitorSchema.parse(jsonData);
+  const result = monitorSchema.safeParse(jsonData);
+
+  if (!result.success) {
+    throw new Error("Invalid response body");
+  }
+
+  const { url, cronTimestamp } = result.data;
+
```

Comment:
Changed to `safeParse` instead of `parse` to validated the response body and throw an error in case.